doc/source/parameters/cfd/velocity_source.rst

```diff
@@ -40,3 +40,6 @@ A Darcy-like source term can be added to the simulation using the following parm
   end
 
 * The ``Darcy type`` parameter specifies the type of Darcy penalization term to be applied to the Navier-Stokes equations. The options are ``none`` or ``phase_change``. The ``phase_change`` model uses the values of the ``Darcy penalty liquid``  and ``Darcy penalty solid`` set-up within the ``phase change`` subsection of the :doc:`physical_properties`.
```

Comment:
I can't find the ``Darcy penalty liquid`` and ``Darcy penalty solid`` in the Physical Properties page

Reply:
I'll add a section recapping the `phase change` subsection!